config/webpacker.yml

```diff
@@ -33,6 +33,7 @@ development:
   <<: *default
   compile: true
   webpack_compile_output: true
+  check_yarn_integrity: false
```

Comment:
Webpacker has officially removed yarn integrity since v5, and they recommend to disable it if causing problems.
See [PR](https://github.com/rails/webpacker/pull/2518)
And issues https://github.com/rails/webpacker/issues/2354, https://github.com/rails/webpacker/issues/2517
We are still using v4, but I think we can remove it.
WDYT @didierofrivia @hallelujah ?

Reply:
I think so too. Let's disable it.